qiskit/circuit/quantumcircuit.py

```diff
@@ -489,8 +489,7 @@ def append(self, instruction, qargs=None, cargs=None, label=None):
         the circuit in place. Expands qargs and cargs.
 
         Args:
-            instruction (qiskit.circuit.Instruction or QuantumCircuit or BaseOperator): instruction
-                to append.
+            instruction (Instruction or QuantumCircuit or BaseOperator): instruction to append.
```

Comment:
The `qiskit.circuit.` prefix seems to be needed for Sphinx, since otherwise it does not know whether it is a  Pulse Instruction of circuit Instruction